### ISSUE TITLE
Can't enable "Dynamically scalable" function for VM from ISO

### ISSUE THREAD

**Issue by top-secrett** [apache/cloudstack#9084]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.0.0
~~~

##### SUMMARY

For new features, show how the feature would be used.
-->
When instance created from ISO I can't enable "Dynamically scalable" function, because instance "attached" to a non-existent template with ISO's id

![image](https://github.com/apache/cloudstack/assets/92435079/0944d0b1-537e-4b59-84b0-2aa14dd2894d)

Click on Template Oracle Linux 8.4

![image](https://github.com/apache/cloudstack/assets/92435079/3e68115f-d917-4595-8140-440a1c0777e1)
![image](https://github.com/apache/cloudstack/assets/92435079/0373167f-a83f-419e-b6f5-6a4d7b809bdc)

2024-05-15 17:16:56,771 ERROR [c.c.a.q.QueryManagerImpl] (qtp379645464-4368326:ctx-87c17956 ctx-34df3704) (logid:1a90ee60) Incorrect format of the template id 204
2024-05-15 17:16:56,771 INFO  [c.c.a.ApiServer] (qtp379645464-4368326:ctx-87c17956 ctx-34df3704) (logid:1a90ee60) Incorrect format ISO of the specified template id

**Comment by shwstppr** [comment]
@top-secrett UI is wrongly redirecting ISO ID to templates page. For now, you may click on the ISO link instead.
In 4.19 I don't think UI form has adoption to update dynamicallyscalable flag for  an ISO. Therefore, you will have to use API,
```
update iso id=<ISO_ID> isdynamicallyscalable=true
```

**Comment by shwstppr** [comment]
@top-secrett I've created #9094 and #9092 to address the two issues

**Comment by shwstppr** [comment]
Fixed with #9094
